encoders/nlp/VSETextEncoder/__init__.py

```diff
@@ -0,0 +1,79 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import pickle
+
+from jina.executors.encoders.frameworks import BaseTorchEncoder
+from jina.executors.decorators import as_ndarray, batching
+
+from .model import VSE
+from .vocab import Vocabulary
+
+class CustomUnpickler(pickle.Unpickler):
+
+    def find_class(self, module, name):
+        try:
+            return super().find_class(__name__, name)
+        except AttributeError:
+            return super().find_class(module, name)
+
+
+class VSETextEncoder(BaseTorchEncoder):
+    """
+    :class:`VSETextEncoder` encodes data from a ndarray of texts into a
+        ndarray of `B x D`. using VSE text_emb branch
+
+    """
+
+    def __init__(self, path: str = 'runs/f30k_vse++_vggfull/model_best.pth.tar',
+                 vocab_path: str = 'vocab/f30k_vocab.pkl', *args, **kwargs):
+        """
+        :path : path where to find the model.pth file
+        :vocab_path : path where to find the vocab.pkl file
+        """
+        super().__init__(*args, **kwargs)
+        self.path = path
+        self.vocab_path = vocab_path
```

Comment:
Where the model file is stored?

Reply:
It is downloaded at docker build time. However, it is too big (because the author put almost every single experiment inside the same .tar) I need a place where I can publicly host a 600MB file, wherr could I do that?

Reply:
Same goes for the vocabulary

Reply:
Ah no, it is faster here in github, I think is good to be merged @nan-wang !

We are not uploading any heavy file inside the repo